daily_milestone.go

```diff
@@ -132,6 +184,24 @@ func main() {
 	// Initializing logger
 	LoggerSetup(os.Stdout)
 	// Calling getProjectID
-	getProjectID(APIBase, Token, Project, Namespace)
-	createMilestoneData(Advance)
+	if Token == "" || APIBase == "" || Namespace == "" || Project == "" {
```

Comment:
This check is not necessary in the go implementation as this could be done via the flag library and required flags. But For now that's alright as you wanted to port if quite similarly to the python one.

Reply:
Ok.